{
  "name": "on-error-resume-next",
  "version": "1.2.1-0",
  "description": "",
  "main": "lib/index.js",
  "files": [
    "lib/**/*"
  ],
  "scripts": {
    "build": "babel --out-dir lib src --ignore **/*.spec.js,**/*.test.js",
    "coveralls": "cat ./coverage/lcov.info | coveralls",
    "prestart": "npm run build:babel",
    "start": "npm run build:babel -- --skip-initial-build --watch",
    "test": "jest --collectCoverage true"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/compulim/on-error-resume-next.git"
  },
  "author": "William Wong <compulim@hotmail.com> (http://compulim.info/)",
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/compulim/on-error-resume-next/issues"
  },
  "homepage": "https://github.com/compulim/on-error-resume-next#readme",
  "jest": {
    "collectCoverage": true
  },
  "devDependencies": {
    "@babel/cli": "^7.8.4",
    "@babel/core": "^7.9.6",
    "@babel/plugin-proposal-object-rest-spread": "^7.9.6",
    "@babel/preset-env": "^7.9.6",
    "babel-jest": "^26.0.1",
    "babel-plugin-add-module-exports": "^1.0.2",
    "jest": "^26.0.1",
    "regenerator-runtime": "^0.13.5"
  }
}
